contrib/slime-repl.el

```diff
@@ -129,6 +129,19 @@ current repl's (as per slime-output-buffer) window."
 (defvar slime-repl-input-start-mark)
 (defvar slime-repl-prompt-start-mark)
 
+(defvar slime-repl-history-use-mark 'T
+  "When T, use the mark as starting point for history lookup and replacement.")
+
+(defun slime-repl-history-yank-start ()
+  "The position of the current input which
+  slime-repl-previous-input will replace from.  When
+  slime-repl-history-yank-start is T, and (mark) is after the
+  current input start, return it.  Otherwise, return
+  slime-repl-input-start-mark."
```

Comment:
Don't indent the docstring, use quotes around the functions you're referencing, and write T in downcase. (Or non-null, which sounds a bit too technical IMHO but is the idiomatic Emacs way.) See <https://www.gnu.org/software/emacs/manual/html_node/elisp/Documentation-Tips.html> for many more details.